client/state/stats/lists/reducer.js

```diff
@@ -21,33 +21,44 @@ import {
 
 /**
  * Returns the updated requests state after an action has been dispatched. The
- * state maps site ID, post ID and stat keys to whether a request is in progress.
+ * state maps site ID, post ID and stat keys to whether a request status.
```

Comment:
should this read: "state maps site ID, post ID and stat keys to the request stats"?